giskard/ml_worker/testing/stat_utils.py

```diff
@@ -14,7 +15,10 @@ def paired_t_test(population_1, population_2, alternative, critical_quantile):
     if alternative not in ["less", "greater"]:
         raise ValueError("Incorrect alternative hypothesis! It has to be one of the following: ['less', 'greater']")
 
-    stat, p_value = stats.ttest_rel(population_1, population_2, alternative=alternative)
+    if np.array_equal(population_1, population_2):
+        p_value = 1.
+    else:
+        stat, p_value = stats.ttest_rel(population_1, population_2, alternative=alternative)
```

Comment:
```suggestion
        _, p_value = stats.ttest_rel(population_1, population_2, alternative=alternative)
```

it's common to replace unused variables with `_`